fds/run.py

```diff
@@ -161,29 +167,43 @@ def __init__(self, arguments: dict):
         )
 
     def execute(self):
-        # Run pre execute hook
-        hook_ret_code = self.hooks_runner.run()
+        arguments = self.arguments
+        self.logger.debug(f"arguments passed: {arguments}")
+
+        hook_ret_code = self.hooks_runner.run(self.hooks_runner.update_check)
         if hook_ret_code != 0:
             return hook_ret_code
+        # No need to run any hooks
+        if arguments.get(Commands.VERSION.value):
+            self.service.version()
+            return 0
```

Comment:
This should probably run even before the update check hook, I think

Reply:
Consider this is like about section (to see version), typically products show updates in about section. That was my reasoning. Do you still want to move this before update check?

Reply:
Hmm I don't know, the way I think about it is that if the user is running `fds version` they just want to see the version, maybe in a script. But I guess that we're currently not very script friendly in general so it could make sense.

However, I think it would still make the most sense to first display the current version and only afterwards suggest to upgrade.